### DISCUSSION TITLE
Add some Files to Use in Stirling?

### DISCUSSION THREAD

**Discussion by Lucifor1976** [Stirling-Tools/Stirling-PDF#1964] [General]
Is it possible to make frequently used files such as stamps or signatures directly selectable or stored in Stirling? Manually selecting the files every time is quite tedious.

**Comment by Frooodle** [comment]
This is now possible

Reuse Stored Files
Certain functionality like Sign supports pre-saved files stored at /customFiles/signatures/. Image files placed within here will be accessible to be used via the web UI. Currently, this supports two folder types:

/customFiles/signatures/ALL_USERS: Accessible to all users, useful for organizations where many users use the same files or for users not using authentication

/customFiles/signatures/{username}: Such as /customFiles/signatures/froodle, accessible only to the froodle username, private for all others
